### PR TITLE
Adhoc: Added Upgrade to Provisioner Sanity test execution

### PR DESCRIPTION
**Pull request by aayushisharma88**
Signed-off-by: root <aayushi.sharma@seagate.com>

# Problem Statement
- This is for Provisioner Upgrade Sanity.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [x] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] https://eos-jenkins.colo.seagate.com/job/Provisioner/job/cortx-rgw-cluster-upgrade_prov/19/console
- [ ] Jenkins pipelines used for testing

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
- Please add Jira ticket details in PR title
- Please fill details in PR template
- Please fix Codacy issues

File: jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy [hunk 1]
```diff
@@ -52,6 +52,8 @@ pipeline {
 
         string(name: 'CORTX_SCRIPTS_REPO', defaultValue: 'https://github.com/Seagate/cortx-k8s', description: 'Repository for cortx-k8s scripts (Services Team)', trim: true)
         string(name: 'CORTX_SCRIPTS_BRANCH', defaultValue: 'cortx-test', description: 'cortx-k8s scripts (Provisioner Team)', trim: true)
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'https://github.com/aayushisharma88/cortx-re', description: 'Repository for cortx-re scripts (Services Team)', trim: true)
```
Comment (gauravchaudhari02) [inline jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy:55]: please change repository to `Seagate/cortx-re`

File: jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy [hunk 2]
```diff
@@ -52,6 +52,8 @@ pipeline {
...
+        string(name: 'CORTX_RE_REPO', defaultValue: 'UPGRADE_SANITY', description: 'Repository for cortx-re scripts (Services Team)', trim: true)
```
Comment (gauravchaudhari02) [inline jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy:56]: please change branch to `main`

**Review by shailesh-vaidya — APPROVED** [review]

**Review by gauravchaudhari02 — APPROVED** [review]

File: jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy (hunk 1 above)
Comment (shailesh-vaidya) [reply jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy]: @gauravchaudhari02 , @aayushisharma88  - Can you please resolve this thread

File: jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy (hunk 2 above)
Comment (shailesh-vaidya) [reply jenkins/automation/kubernetes/nightly-prvsn-sanity-deploy.groovy]: @gauravchaudhari02 , @aayushisharma88  - Can you please resolve this thread

**Pull request merged**